deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -505,6 +530,8 @@ def merge_input_provider(index, item):
         hash_bucket_count=params.hash_bucket_count,
         hb_index_to_entry_range=hb_id_to_entry_indices_range,
         compactor_version=CompactorVersion.V2.value,
+        input_inflation=input_inflation,
```

Comment:
Should we also include the output (materialized) inflation / average record sizes?

Reply:
I did not include it because that can be derived from pyarrow write result.